cmd/transportRequestCreateSOLMAN.go

```diff
@@ -0,0 +1,85 @@
+package cmd
+
+import (
+	"github.com/SAP/jenkins-library/pkg/command"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/telemetry"
+	"github.com/SAP/jenkins-library/pkg/transportrequest/solman"
+)
+
+type transportRequestCreateSOLMANUtils interface {
+	command.ExecRunner
```

Comment:
use solman.Exec from common.go

Reply:
This is generated coding. I would like to suggest not to modify the generated coding. Using the suggested interface would not be wrong, though. This interface defines more functions than we need, but also all functions we need.